cisco_aci/datadog_checks/cisco_aci/models.py

```diff
@@ -0,0 +1,148 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import six
+
+if six.PY3:
+    from typing import Optional
+
+    from pydantic import BaseModel, ConfigDict, Field, computed_field, field_validator
+
+    class NodeAttributes(BaseModel):
+        address: Optional[str] = None
+        fabric_st: Optional[str] = Field(default=None, alias="fabricSt")
+        role: Optional[str] = None
+        dn: Optional[str] = None
+        model: Optional[str] = None
+        version: Optional[str] = None
+        serial: Optional[str] = None
+        vendor: Optional[str] = Field(default='cisco-aci')
+        namespace: Optional[str] = Field(default='default')
+
+        @computed_field
+        @property
+        def device_type(self) -> str:
+            if self.role in ['leaf', 'spine']:
+                return 'switch'
+            if self.role in ['controller', 'vleaf', 'vip', 'protection-chain']:
```

Comment:
cisco_aci is not a valid device_type today. I'm not sure we have any existing device type today that can map to these type of devices. So maybe we should have `other` for all of them?

cc @NouhaManai96, would love your opinion on that one

Reply:
I agree that we should use `other` in this case as `cisco_aci` is not an actual device type
(FYI the list of valid `device_types` for now can be found [here](https://github.com/DataDog/datadog-agent/blob/18d6c0b82c6c8246e6f2001417ee7bff38088740/pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go#L36-L50))
I would also suggest adding the `self.role` as a tag `role` to not lose this information that can be useful to users, we do a similar thing with cisco sdwan with the `type` tag

Reply:
yes for sure! role is currently added as a tag, bit more specifically as `apic_role` and `switch_role` in the tags for the device metadata 

and will do! will update to use `other` instead!